test/inductor/test_torchinductor.py

```diff
@@ -6182,6 +6182,7 @@ def fn(a):
         opt_fn = torch._dynamo.optimize("inductor")(fn)
         same(fn(x), opt_fn(x))
 
+    @unittest.skipIf(not HAS_CPU, "requires C++ compiler")
```

Comment:
@zhuhaozhe Weird that it has problems on other devices. The test should be general?

Reply:
Hi, @williamwen42. May you help to give more info about how to reproduce?
I cannot open https://www.internalfb.com/login/?next=https%3A%2F%2Fwww.internalfb.com%2Fintern%2Ftest%2F844425024008760%2F.

From my understanding the only backend-related code is 
https://github.com/pytorch/pytorch/blob/a1118a9bfaf28a5b1727f3725922067b513ca1df/test/inductor/test_torchinductor.py#L6236
but I am not sure whether this should be an issue.

Reply:
This one is an issue with gcc not being installed on some machines.  Not sure there is something to fix.